.github/workflows/lint.yml

```diff
@@ -24,3 +24,22 @@ jobs:
         with:
           python-version: "3.x"
       - uses: pre-commit/action@v3.0.0
+
+  patchcheck:
+    if: "github.repository == 'python/cpython' && github.event_name == 'pull_request'"
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - name: "Run patchcheck"
+        run: |
+          git status && git branch
```

Comment:
Why not putting them on two lines?

Reply:
Should we make this a distinct step for "git info"? I'm unsure what it achieves, though.